.github/workflows/msbuild.yml

```diff
@@ -35,15 +50,6 @@ jobs:
       if: matrix.os == 'ubuntu-16.04'
       run: |
         sudo apt-get install -y build-essential
-    - name: Checkout Dafny
-      uses: actions/checkout@v2
-      with:
-        submodules: recursive
-        path: dafny
-    - name: Setup dotnet
```

Comment:
How is dotnet installed now?

Reply:
With a duplicate dotnet setup instruction elsewhere in the script

Reply:
Can you point me to it? I couldn't find it.